furusiya/view/adapter/tdl_adapter.py

```diff
@@ -47,14 +53,72 @@ def get_input():
         """
         Ask for input. If none, returns None.
         """
-        keypress = False
         for event in tdl.event.get():
-            if event.type == 'KEYDOWN':
-                print("KEY DOWN: {0}".format(event))
-                return event  # contains key pressed
-            if event.type == 'MOUSEMOTION':
-                print("MOUSE: {0}".format(event))
-                return event.cell
-    
-        if not keypress:
-            return None
+            return event
+
+    @staticmethod
+    def toggle_fullscreen():
+        tdl.set_fullscreen(not tdl.get_fullscreen())
+
+    @staticmethod
+    def event_closed():
+        return tdl.event.is_window_closed()
+
+    def draw_string(self, x, y, string, color):
+        self.root.draw_str(x, y, string, bg=None, fg=color)
+
+    def create_menu(self, header, options, width):
```

Comment:
This shouldn't be here. This is logic, not actual wrapper info. But, it does use `window.draw_rect` and `self.root.blit`. I'm not sure what to do with these, we could extract them into methods, eg. `def blit() # calls self.root.blit` or `def blit(root) # calls root.blit`. Not sure.  Let's think more on this.